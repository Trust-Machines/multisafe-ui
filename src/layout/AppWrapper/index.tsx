import React from 'react';
import {Box} from '@mui/material';

const AppWrapper = (props: { children: React.ReactNode }) => {
    const boxStyles = {
        position: 'absolute',
        left: 0,
        right: 0,
        top: 0,
        bottom: 0,
        width: '100%',
        height: '100%',
        display: 'flex',
        flexDirection: 'column'
    }

    return <Box sx={boxStyles}>{props.children}</Box>
}

export default AppWrapper;