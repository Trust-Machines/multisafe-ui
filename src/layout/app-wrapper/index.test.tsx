import {render} from '@testing-library/react';

import AppWrapper from './index';

test('1 Default render', () => {
    const view = render(<AppWrapper><span>children</span></AppWrapper>);
    expect(view.container).toMatchSnapshot()
});