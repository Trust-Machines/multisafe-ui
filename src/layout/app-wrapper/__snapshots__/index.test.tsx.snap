// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <div
    class="MuiBox-root css-vbk5xt"
  >
    <span>
      children
    </span>
  </div>
</div>
`;
